keyvault/client/src/main/java/com/azure/keyvault/models/Secret.java

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.keyvault.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+
+/**
+ * A secret consisting of a value, id and its attributes.
+ */
+public class Secret extends SecretInfo {
+
+    /**
+     * The secret value.
+     */
+    @JsonProperty(value = "value")
+    private String value;
+
+
+    public Secret() {
+
+    }
+
+    public Secret(String secretName, String value) {
+        withName(secretName);
+        this.value = value;
+    }
+
+    /**
+     * Get the value value.
+     *
+     * @return the value value
+     */
+    public String value() {
+        return this.value;
+    }
+
+    /**
+     * Set the value value.
+     *
+     * @param value the value value to set
+     * @return the Secret object itself.
+     */
+    public Secret withValue(String value) {
+        this.value = value;
+        return this;
+    }
```

Comment:
This accessor should be removed.  Once the value of a secret is set, it cannot be updated.  The model should reflect this.

Reply:
Agreed. Removed.